api/dp-openapi.yaml

```diff
@@ -690,6 +690,160 @@ paths:
         default:
           $ref: '#/components/responses/UnexpectedError'
 
+  /queues/{roundid}:
+    get:
+      tags:
+      - queues
+      description: Gets the release queues for a round
+      parameters:
+      - name: roundid
+        in: path
+        description: Round ID
+        required: true
+        schema:
+          type: string
+      - name: field
+        description: Queue field to return, if not set all fields are returned.
+          To request multiple fields, append with [] and pass one per desired
+          field (e.g. field[]=projectid&field[]=author).
+        in: query
+        schema:
+          type: string
+      - name: show
+        description: Which queues to show.
```

Comment:
We should confirm how this is rendered as I think it's just going to all run together as-is. IIRC we use HTML in other descriptions to force formatting.